app.js

```diff
@@ -298,7 +298,7 @@ const map = sitemap({
     }
 });
 
-if (NODE_ENV === 'production') {
+if (NODE_ENV === 'production' || NODE_ENV === 'test') {
```

Comment:
Perhaps just always render the site map? Is there a reason to have it not generated in dev?